account_invoice_analysis_analytic/__openerp__.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Authors: Guewen Baconnier, Leonardo Pistone, Jordi Ballester
+#    Copyright 2015 Camptocamp SA
+#    Copyright 2015 Eficent
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+
+{
+    "name": "Invoices Analysis by analytic account",
+    "version": "0.1",
+    "depends": ['account', 'analytic'],
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Accounting & Finance",
+    'description': """
+Invoices Analysis by analytic account
+=====================================
+
+This module is meant to extend the report 'Invoices Analysis' adding the
+analytic account
+
+
+Usage
+=====
+
+Go to 'Reporting | Accounting | Invoices Analysis'. It is possible to
+group by or filter by analytic account.
+
+
+Known issues / Roadmap
+======================
+
+Credits
+=======
+
```

Comment:
ADD Bug tracker like in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst